components/dropdown/dropdown-content-mixin.js

```diff
@@ -320,6 +320,15 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		this.addEventListener('d2l-dropdown-position', this.__toggleScrollStyles);
 	}
 
+	async getUpdateComplete() {
+		const fontsPromise = document.fonts ? document.fonts.ready : Promise.resolve();
+		await super.getUpdateComplete();
+		/* wait for the fonts to load because browsers have a font block period
+		where they will render an invisible fallback font face that may result in
+		improper width calculations before the real font is loaded */
+		await fontsPromise;
+	}
```

Comment:
I found that if the page loaded in an `opened` state then the positioning wasn't right. This sometimes affected visual diffs as well. I found that waiting for the fonts to be ready fixed this, and this is the pattern that tooltip uses so I copied this here.

Reply:
Is there a case for this to be moved up to a parent class?